src/lang-promql/complete/promql.terms.ts

```diff
@@ -17,67 +19,251 @@ export const binOpTerms = [
+    info: 'Calculate base-2 logarithm of input series',
+  },
+  {
+    label: 'max_over_time',
+    info: 'Return the maximum value over time for input series',
+  },
+  {
+    label: 'min_over_time',
+    info: 'Return the minimum value over time for input series',
+  },
+  {
+    label: 'minute',
+    info: 'Return the minute of the hour for provided timestamps',
+  },
+  {
+    label: 'month',
+    info: 'Return the month for provided timestamps',
+  },
+  {
+    label: 'predict_linear',
+    info: 'Predict the value of a gauge into the future',
+  },
+  {
+    label: 'quantile_over_time',
+    info: 'Calculate value quantiles over time for input series',
+  },
+  {
+    label: 'rate',
+    info: 'Calculate per-second increase over a range vector (for counters)',
+  },
+  {
+    label: 'resets',
+    info: 'Return number of value decreases (resets) in input series of time',
+  },
+  {
+    label: 'round',
+    info: 'Round values of input series to nearest integer',
+  },
+  {
+    label: 'scalar',
+    info: 'Convert single-element series vector into scalar value',
+  },
+  {
+    label: 'sort',
+    info: 'Sort input series ascendingly by value',
+  },
+  {
+    label: 'sort_desc',
+    info: 'Sort input series descendingly by value',
+  },
+  {
+    label: 'sqrt',
+    info: 'Return the square root for input series',
+  },
+  {
+    label: 'stddev_over_time',
+    info: 'Calculate the standard deviation within input series over time',
+  },
+  {
+    label: 'stdvar_over_time',
+    info: 'Calculate the standard variation within input series over time',
+  },
+  {
+    label: 'sum_over_time',
+    info: 'Calculate the sum over the values of input series over time',
+  },
+  {
+    label: 'time',
+    info: 'Return the Unix timestamp at the current evaluation time',
+  },
+  {
+    label: 'timestamp',
+    info: 'Return the Unix timestamp for the samples in the input vector',
+  },
+  {
+    label: 'vector',
+    info: 'Convert a scalar value into a single-element series vector',
+  },
+  {
+    label: 'year',
+    info: 'Return the year for provided timestamps',
+  },
 ];
-export const aggregateOpTerms = [
-  { label: 'avg' },
-  { label: 'bottomk' },
-  { label: 'count' },
-  { label: 'count_values' },
-  { label: 'group' },
-  { label: 'max' },
-  { label: 'min' },
-  { label: 'quantile' },
-  { label: 'stddev' },
-  { label: 'stdvar' },
-  { label: 'sum' },
-  { label: 'topk' },
+functionIdentifierTerms.forEach((term) => {
+  term.detail = 'function';
+});
+export const aggregateOpTerms: AutoCompleteNode[] = [
+  {
+    label: 'avg',
+    info: 'Calculate the average over dimensions',
+  },
+  {
+    label: 'bottomk',
+    info: 'Smallest k elements by sample value',
+  },
+  {
+    label: 'count',
+    info: 'Count number of elements in the vector',
+  },
+  {
+    label: 'count_values',
+    info: 'Count number of elements with the same value',
+  },
+  {
+    label: 'group',
+    info: 'All values in the resulting vector are 1',
```

Comment:
While this is true, it misses the main purpose of the `group` aggregator, which is not the sample value, but the grouping. How about:


```suggestion
    info: 'Group series, while setting the sample value to 1'',
```